samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages.properties

```diff
@@ -339,6 +339,30 @@ secure_delivery_exit_pwd = Exit password (optional)
 exit_password_error = The only characters allowed for the exit password are a-z, A-Z and 0-9
 none_secure_delivery_module = None
 
+# Safe Exam browser support
+seb_config_mode=How do you want to confiure SEB?
+seb_config_mode_client=Use client configuration
+seb_config_mode_manual=Configure Manually
+seb_config_mode_upload=Upload my own config
+seb_allow_user_quit_seb=Enable exiting from SEB
+seb_allow_reload_in_exam=Enable reload in exam
+seb_show_taskbar=Show SEB task bar
+seb_show_reload_button=Show reload button
+seb_show_time=Show time
+seb_show_keyboard_layout=Show keyboard layout
+seb_show_wifi_control=Show Wi-Fi control
+seb_allow_audio_control=Enable audio controls
+seb_allow_spell_checking=Enable spell checking
+seb_config_upload=SEB config file
+seb_config_upload_success=SEB config file was uploaded successfully.
+seb_config_upload_error=SEB config file coule not be uploaded. Please make sure the configuration is an .seb \
+file that is not encrypted.
+seb_exam_keys=Allowed browser exam keys
+seb_exam_keys_upload_info=For uploaded configurations the exam keys can only be set, after the assessment is \
+published. You will need to download the (altered) configuration again and insert the exam keys based on that file.
+seb_exam_keys_upload_published_info=Please insert exam keys based on the config file that you can download above. \
+The configuration has been altered, when the assessment was published.
```

Comment:
Please, provide the same strings to the _ES, _CA and _EU files, I'll help with the translation.

Reply:
Add the S2U-21 code and put it at the end of the file.